plugins/woocommerce/package.json

```diff
@@ -445,34 +445,34 @@
 					}
 				},
 				{
-					"name": "Core API tests",
+					"name": "Core API tests (HPOS:on)",
```

Comment:
Changes (Core API tests (HPOS:on)):
- do not track patterns/templates changes in PRs as those unlikely to impact REST (trunk push will cover us)
- changes in other packages are irrelevant, except  one exception